.github/workflows/project-automation.yml

```
@@ -10,6 +10,21 @@ on:
   issue_comment:
     types: [created]
 
+permissions:
+  contents: read
+  issues: write
+  pull-requests: read
+  discussions: write
+  projects: write
```

actionlint (1.7.7)

unknown permission scope "projects". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)